src/app.js

```diff
@@ -162,7 +165,7 @@ export const start = args => {
       args = args.join('/');
     }
 
-    screen.spawn('git', ['log', args, '--color=always']);
+    screen.spawn('git', ['log', args, gitLogArguments]);
```

Comment:
CIO is breaking here -- I added a string of arguments to the custom log arguments in my `check-it-out.json` and I am unable to call git log with `[spacebar]`.

Take a look at this function and how the arguments are generated. we may need to change how we format the log arguments as well as add them to the process spawn.

Reply:
Weird, my argument works just fine:
`--pretty=format:'%C(Yellow) %h %Cred%ad %Cblue%an %Creset%s%Cgreen%d'`

What is yours?

Reply:
I tried one of my git log aliases: `--graph --abbrev-commit --decorate --date=relative --format=format:'%C(bold blue)%h%C(reset) - %C(bold green)(%ar)%C(reset) %C(white)%s%C(reset) %C(dim white)- %an%C(reset)%C(bold yellow)%d%C(reset)' --all`

Reply:
https://github.com/chjj/blessed#methods-1 Spawn method looks like arguments actually should be a part of the 2nd argument. Not sure why your arguments are passed in correctly.

Reply:
Bummer. Multiple arguments seem to be the issue. Anything with only one argument works

Reply:
If that's the case, can we try changing the config to contain an array of arguments and just spread or join them when we call spawn?

Reply:
Did a quick test, passed your 6 log args from config into logArgs as string and spawned with logArgs but didn't work either. I can look into that desperately tomorrow, if you find a solution please let me know. Take care!